src/Raygun4php/interfaces/RaygunMessageInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Raygun4php\Interfaces;
+
+interface RaygunMessageInterface
+{
+    public function build(\Exception $exception): void;
+
+    public function toJson(): string;
```

Comment:
I think I've mentioned this before, but perhaps we should use JsonSerializable here?

Reply:
I mentioned in #116 my reasoning behind `toJson()` is to not have to deal with the encoding issus solved by `toJsonRemoveUnicodeSequences()` and `removeNullBytes()`. 
It is at mather of taste, but I like it when an object takes care of all aspects of serializing itself.